packages/ast-spec/tsconfig.build.json

```diff
@@ -1,12 +1,19 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "composite": true,
-    "jsx": "preserve",
-    "outDir": "./dist",
-    "rootDir": "./src",
-    "resolveJsonModule": true
+    "baseUrl": ".",
+    "rootDir": "src",
+    "outDir": "dist",
+    "tsBuildInfoFile": "dist/tsconfig.build.tsbuildinfo",
```

Comment:
```suggestion
```
could possibly reduce some boilerplate by putting this:

```json
    "outDir": "${configDir}/dist",
    "tsBuildInfoFile": "${configDir}/dist/tsconfig.build.tsbuildinfo",
```
into

`../../tsconfig.base.json`